app/services/handover_date_service.rb

```diff
@@ -18,19 +20,32 @@ def self.handover(offender)
 
   def self.nps_start_date(offender)
     if offender.early_allocation?
-      early_allocation_handover_date(offender)
+      offender.conditional_release_date - 18.months
     elsif offender.indeterminate_sentence?
-      offender.earliest_release_date - 8.months
+      offender.tariff_date - 8.months # earliest_release_date
+    elsif offender.parole_eligibility_date.present?
+      offender.parole_eligibility_date - 8.months
     else
-      offender.earliest_release_date - (7.months + 15.days)
+      [
+        offender.conditional_release_date,
+        offender.automatic_release_date
+      ].compact.min - (7.months + 15.days)
     end
   end
 
   def self.crc_handover_date(offender)
-    [
-        offender.home_detention_curfew_eligibility_date,
-        offender.conditional_release_date
+    crd_ard = [
+        offender.conditional_release_date,
+        offender.automatic_release_date
     ].compact.map { |date| date - 12.weeks }.min
+
+    if offender.home_detention_curfew_actual_date.present?
```

Comment:
whats the difference between `home_detention_curfew_eligibility_date` and `home_detention_curfew_actual_date`. When would one be entered over the other?